src/data/markdown/translated-guides/en/04 Results visualization/02 Cloud.md

```diff
@@ -61,15 +69,65 @@ execution: local
 
 ![k6 Cloud Test Results](./images/Cloud/k6-cloud-results.png)
 
-> When you send the results to the k6 Cloud, data will be continuously streamed to the cloud. While this happens the state of the test run will be marked as `Running`. A test run that ran its course will be marked `Finished`. The run state has nothing to do with the test passing any thresholds, only that the test itself is operating correctly.
-> 
-> If you deliberately abort your test (e.g. by pressing _Ctrl-C_), it will still be considered `Finished`. You can still look and analyze the test data you streamed so far. The test will just have run shorter than originally planned.
-> 
-> Another possibility would be if you lose network connection with the k6 Cloud while your test is running. In that case the k6 Cloud will patiently wait for you to reconnect. In the meanwhile your test's run state will continue to appear as `Running` on the web app.
-> 
-> If no reconnection happens, the k6 Cloud will time out after two minutes of no data, setting the run state to `Timed out`. You can still analyze a timed out test but you'll of course only have access to as much data as was streamed before the network issue.
+When you send the results to the k6 Cloud with `k6-out`, data will be
+continuously streamed to the cloud. While this happens the state of the test run
+will be marked as `Running`. A test run that ran its course will be marked
+`Finished`. The run state has nothing to do with the test passing any
+thresholds, only that the test itself is operating correctly.
+
+If you deliberately abort your test (e.g. by pressing _Ctrl-C_), your test will
+end up as `Aborted by User`. You can still look and analyze the test data you
+streamed so far. The test will just have run shorter than originally planned.
+
+Another possibility would be if you lose network connection with the k6 Cloud
+while your test is running. In that case the k6 Cloud will patiently wait for
+you to reconnect. In the meanwhile your test's run state will continue to
+appear as `Running` on the web app.
+
+If no reconnection happens, the k6 Cloud will time out after two minutes of no
+data, setting the run state to `Timed out`. You can still analyze a timed out
+test but you'll of course only have access to as much data as was streamed
+before the network issue.
+
+# Advanced settings
```

Comment:
@Griatch  H2`##` will be shown on the Table Of Content

Reply:
Should I change to ## here then?

Reply:
Yes. Just refresh the page and you will see how it appears on the ToC (right sidebar).

Reply:
I ... don't see any changes in the ToC at all when building this locally. But I changed this to ## now.